skimage/measure/simple_metrics.py

```diff
@@ -142,3 +145,52 @@ def compare_psnr(im_true, im_test, data_range=None):
 
     err = compare_mse(im_true, im_test)
     return 10 * np.log10((data_range ** 2) / err)
+
+
+def enhancement_measure(image: np.ndarray,
+                        eps: float = 1e-6,
+                        size: int = 3) -> float:
```

Comment:
Please, keep a consistent arg order in the signature and the docstring.

Reply:
@sciunto Thank you for pointing that out, I fixed it.